grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovement.groovy

```diff
@@ -229,6 +230,19 @@ class OutboundStockMovement implements Serializable, Validateable {
         return shipment?.currentStatus == ShipmentStatusCode.RECEIVED
     }
 
+    @Deprecated
```

Comment:
can you add a comment here discribing why we're deprecating this and what to use instead? It's useful for future us to remember.

Reply:
I'm also wondering why we actually want to deprecate the method we've just created? Did I miss something?

Reply:
Unless I misunderstood our conversation on the tech-huddle I remember you @jmiranda asking for it to be deprecated out of the gate since this is kind of a "workaround" before the actual refactor for statuses and to prevent developers from relying on this method.

I agree, my bad for not including any comment for it.